src/main/java/com/brandon3055/draconicevolution/client/gui/GUIParticleGenerator.java

```diff
@@ -137,707 +259,240 @@ else if (page == 10) {
         fontRendererObj.drawStringWithShadow("Ctrl +- 50", posX + 215, posY + 31, 0xFFFFFF);
         fontRendererObj.drawStringWithShadow("Shift+Ctrl +- 100", posX + 215, posY + 41, 0xFFFFFF);
 
-        super.drawScreen(x, y, f);
-    }
-    // @formatter:on
-
-    @Override
-    protected void mouseClicked(int x, int y, int button) {
-        super.mouseClicked(x, y, button);
+        super.drawScreen(x, y, partialTicks);
     }
 
     @Override
+    @SuppressWarnings("unchecked")
     public void initGui() {
         int posX = (this.width - xSize) / 2;
         int posY = (this.height - ySize) / 2;
         buttonList.clear();
 
-        if (page < 10) {
-            buttonList.add(new GuiButton(33, posX + 213, posY + 177, 47, 20, "===>"));
-            buttonList.add(new GuiButton(32, posX - 47, posY + 177, 47, 20, "<==="));
+        // Navigation
+        buttonList.add(previousPage = new GuiButton(100, posX - 47, posY + 177, 47, 20, "<==="));
```

Comment:
Those arrows are pretty ugly. I think we should use just `<`/`>` like in NEI for switching between pages.